ibm/service/power/ibm_pi_constants.go

```diff
@@ -4,6 +4,9 @@ import "time"
 
 const (
 	// Arguments
+	Arg_NetworkSecurityGroupID              = "pi_network_security_group_id"
+	Arg_UserTags                            = "pi_user_tags"
+	Arg_IPAddress                           = "pi_ip_address"
```

Comment:
These need to be put into alphabetical order

Reply:
Updated